src/pages/en/reference/adapter-reference.md

```diff
@@ -42,15 +42,19 @@ The object passed into `setAdapter` is defined as:
 interface AstroAdapter {
 	name: string;
 	serverEntrypoint?: string;
+	shim?: string[];
 	exports?: string[];
+	args?: any;
 }
 ```
 
 The properties are:
 
 * __name__: A unique name for your adapter, used for logging.
 * __serverEntrypoint__: The entrypoint for server-side rendering.
+* __shim__: Optional array of import specifiers pointing to polyfills/shims.
 * __exports__: An array of named exports when used in conjunction with `createExports` (explained below).
+* __args__: Optional args to pass to your server handler.
```

Comment:
If it's correct to say "arguments" in the property description, then I would suggest using the unabbreviated word here.

Reply:
The `args` property already existed, it was just undocumented. Changing it to `arguments` now would be a breaking change.

Reply:
I think Sarah means in the explanation of the option (rather than changing the property name).

Reply:
```suggestion
* __args__: Optional arguments to pass to your server handler.
```